apps/storybook/.storybook/preview.tsx

```diff
@@ -137,12 +138,30 @@ const components = {
 
 const preview: Preview = {
   tags: ['a11y-test'],
+  globalTypes: {
+    html: {
+      description: '"Show code" will output raw html',
+      toolbar: {
+        title: 'Show code as html',
+        icon: 'markup',
+        items: ['true', 'false'],
+        dynamicTitle: false,
+      },
```

Comment:
```suggestion
      toolbar: {
        title: 'Show code as',
        icon: 'markup',
        items: ['html', 'jsx'],
        dynamicTitle: false,
      },
```
Just a suggestion if we need to have it as a dropdown and not a switch